File: ABCPharmaMedicineTrackerSPA/src/app/model/medicine.ts
export interface IMedicine{
    name:string
    brand : string
    price : number 
    quantity : number
    expirdate : Date
    notes : string
}

export class Medicine implements IMedicine{
    name: string;
    brand: string;
    price: number;
    quantity: number;
    expirdate: Date;
    notes: string;

}

File: ABCPharmaMedicineTracker.API/Model/Medicine.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ABCPharmaMedicineTracker.API.Model
{
    public class Medicine
    {
        public Medicine()
        {
        }

        public string Name { get; set; }
        public string Brand { get; set; }
        public decimal Price { get; set; }
        public int Quantity { get; set; }
        public DateTime ExpiryDate { get; set; }
        public string Notes { get; set; }
    }

   

}
